apps/cms/pages/models.py

```diff
@@ -15,31 +19,80 @@ class HomePage(Page):
         verbose_name="Header Image",
         help_text="The Image that is shown on top of the page"
     )
-    subtitle = models.CharField(
+
```

Comment:
Is it possible to make the title also translatable?

Reply:
`title` comes as a required field from wagtail, we could add a german translation but can not rename it. I think its not used anywhere in the template.